### DISCUSSION TITLE
Whether the ForwardRequestProcessors are being used？

### DISCUSSION THREAD

**Discussion by socutes** [apache/rocketmq#6274] [Polls]
org.apache.rocketmq.broker.processor.ForwardRequestProcessor in module Broker, does not appear to be used.
Would like to initiate a discussion to see if it can be removed.

**Comment by RongtongJin** [comment]
Hi @socutes , I think this class is useless, you can subbmit a pull request to delete the class.

**Reply by socutes** [reply]
ok
